data/json/mutations/mutations.json

```diff
@@ -9145,5 +9163,37 @@
     "category": [ "CEPHALOPOD" ],
     "prereqs": [ "INT_UP_4" ],
     "threshreq": [ "THRESH_CEPHALOPOD" ]
+  },
+  {
+    "type": "mutation",
+    "id": "VANITY",
+    "name": { "str": "Highly Vain" },
+    "points": -1,
+    "description": "You know you're beautiful, and the world deserves to know it too.  You prefer to show off your stunningly good looks to the world and will be unhappy if you cover up your face.",
+    "category": [ "ALPHA" ],
+    "prereqs": [ "BEAUTIFUL", "BEAUTIFUL2" ]
+  },
+  {
+    "type": "mutation",
+    "id": "PICKYEATER",
+    "name": { "str": "Picky Eater" },
+    "points": -1,
+    "description": "You're a very picky eater.  You refuse to eat any food that tastes bad unless you're starving.",
+    "category": [ "ALPHA" ],
+    "starting_trait": true
+  },
+  {
+    "type": "mutation",
+    "id": "ALPHA_COMPLEX",
+    "name": { "str": "Superiority Complex" },
+    "points": 0,
+    "description": "You can't stand to prostrate yourself before your lessers and hate it when they don't immediately do what you say.  Persuasion is much more difficult, but you're better at lying and intimidating.",
```

Comment:
Ooh now we could add an EOC that checks followers for this trait and randomly causes them to shift towards mutiny.